models/volunteering-request-offer-base.js

```diff
@@ -29,7 +29,7 @@ const VolunteeringRequestOfferBaseSchema = new mongoose.Schema({
     ref: 'Content',
   },
   regions: {
-    type: [String],
+    type: ['SubRegion'],
```

Comment:
Does it work like that?
From a short search (and the little exprience I've had with mongoose), I think it should look like:
```js
regions: {
  type: [
    {
      type: Schema.ObjectId,
      ref: 'Choices'
    }
  ],
  required: true,
},
```